frontend/packages/vsphere-plugin/locales/en/vsphere-plugin.json

```diff
@@ -0,0 +1,54 @@
+{
+  "vSphere connection": "vSphere connection",
+  "vSphere configurations": "vSphere configurations",
+  "Loading vSphere connection status...": "Loading vSphere connection status...",
```

Comment:
I think we could erase the ellipses (...) because ellipses are used when information is omitted, when you cannot fit all words onto a line, or when you remove less relevant information (like in a quote) https://www.patternfly.org/v4/ux-writing/punctuation